jormungandr-integration-tests/src/common/configuration/node_config_model.rs

```diff
@@ -40,7 +40,7 @@ pub struct NodeConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub storage: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub logger: Option<Logger>,
+    pub log: Option<Log>,
     pub rest: Option<Rest>,
     pub peer_2_peer: Peer2Peer,
```

Comment:
Rename this to `p2p` while we are at it? The underscores give me carpal syndrome :)

Reply:
Hmm, this is _test_ config. I meant the YAML naming, of course.

Reply:
Same as in https://github.com/input-output-hk/jormungandr/pull/627#discussion_r302217842